person.rb

```diff
@@ -0,0 +1,22 @@
+# Person class
+class Person
+  attr_accessor :name, :age
+  attr_reader :id
+
+  def initialize(age, name = 'Unknown', parent_permission: true)
```

Comment:
- [ ] Kindly change the `:` to `=` for `parent_permission` or change the way you call the super in order to make your code valid.

Reply:
Hello @gemmen29 I was using parent_permissin = true but it was giving me a linter error, and when I use parent_permission: true I can't change it to false when I instantiate a new person.
What is the other way to call the super so I can try it?

thank you so much